code/game/machinery/portable_turret.dm

```diff
@@ -470,8 +470,9 @@
 
 	if(check_anomalies)	//if it's set to check for xenos/simpleanimals
 		for(var/mob/living/simple_animal/SA in turretview)
-			if(!SA.stat && (!SA.has_unlimited_silicon_privilege || !(faction in SA.faction))) //don't target dead animals or NT maint drones.
-				targets += SA
+			if(SA.stat || faction in SA.faction || SA.has_unlimited_silicon_privilege) //don't target dead animals or NT maint drones.

```

Comment:
yeah i was unsure since the ref doesn't list the priority of `in` for some reason